antora-ui-camel/src/css/header.css

```diff
@@ -225,7 +225,7 @@ body {
     transform: rotate(-45deg);
     width: 0.5em;
     right: 0;
-    top: 40%;
+    top: 4.5px;
```

Comment:
We can't use pixels, we need to use `rem`. There is no 0.5 pixel size on the screen.